ssl-monitor/frontend/src/components/AlertsDisplay.tsx

```
@@ -0,0 +1,247 @@
+  ListItemText,
+  ListItemIcon,
+  IconButton,
+  Chip,
+  Alert as MuiAlert,
+  CircularProgress,
+  Button,
+  Tooltip,
+} from '@mui/material';
+import {
+  Warning,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)